pkg/kubelet/eviction/helpers.go

```diff
@@ -41,10 +41,16 @@ const (
 	message = "The node was low on compute resources."
 	// disk, in bytes.  internal to this module, used to account for local disk usage.
 	resourceDisk api.ResourceName = "disk"
+	// inodes, in bytes. internal to this module, used to account for local disk inode consumption.
```

Comment:
The comment (and the following ones) says the units of inodes are in bytes, which seems incorrect.